internal/services/servicenetworking/client/client.go

```diff
@@ -6,7 +6,7 @@ package client
 import (
 	"fmt"
 
-	servicenetworking_2023_11_01 "github.com/hashicorp/go-azure-sdk/resource-manager/servicenetworking/2023-11-01"
+	servicenetworking_2023_11_01 "github.com/hashicorp/go-azure-sdk/resource-manager/servicenetworking/2025-01-01"
```

Comment:
Could you update this alias to it matches the version?

```suggestion
	servicenetworking_2025_01_01 "github.com/hashicorp/go-azure-sdk/resource-manager/servicenetworking/2025-01-01"
```

Reply:
updated